d2l-input-checkbox.js

```diff
@@ -102,7 +105,7 @@ $_documentContainer.innerHTML = `<dom-module id="d2l-input-checkbox">
 			}
 		</style>
 		<label>
-			<input type="checkbox" aria-label$="[[ariaLabel]]" aria-labelledby$="[[ariaLabelledby]]" checked="{{checked}}" disabled$="[[disabled]]" name$="[[name]]" on-change="_handleChange" on-focus="_handleFocus" value$="[[value]]">
+			<input type="checkbox" class$="[[_getInputClass(tri)]]" aria-label$="[[ariaLabel]]" aria-labelledby$="[[ariaLabelledby]]" checked="{{checked}}" disabled$="[[disabled]]" name$="[[name]]" on-change="_handleChange" on-click="_onClick" on-focus="_handleFocus" value$="[[value]]">
```

Comment:
Looks like the `class$=[[_getInputClass(tri)]]"` is a relic from some experimentation that never made it in?

Reply:
^ thank you! I'll take that out